src/common/Pagination/PageInfo/index.js

```diff
@@ -1,29 +1,51 @@
 import React from 'react'
 import { useSelector } from 'react-redux';
 import { selectMovies } from '../../../features/movies/moviesSlice';
+import { selectPopularPeople } from '../../../features/people/popularPeopleSlice';
 import { PageCaption, Paragraph } from './styled';
 
 const PageInfo = ({ type }) => {
 
     const movies = useSelector(selectMovies);
+    const people = useSelector(selectPopularPeople);
 
     return (
+
         <Paragraph>
-            <PageCaption>
-                Page
-            </PageCaption>
-            <PageCaption number>
-                {type === "movies" && movies.page}
-                {/* {type === "people" && people?.page} */}
-            </PageCaption>
-            <PageCaption>
-                of
-            </PageCaption>
-            <PageCaption number>
-                {type === "movies" && movies.total_pages}
-                {/* {type === "people" && people?.total_pages} */}
-            </PageCaption>
+            { movies && type === "movies" &&
+                <>
+                    <PageCaption>
+                        Page
+                    </PageCaption>
+                    <PageCaption number>
+                        {movies.page}
+                    </PageCaption>
+                    <PageCaption>
+                        of
+                    </PageCaption>
+                    <PageCaption number>
+                        {movies.total_pages}
+                    </PageCaption>
+                </>
+            }
+            { people && type === "people" &&
+                <>
+                    <PageCaption>
+                        Page
+                    </PageCaption>
+                    <PageCaption number>
+                        {people.page}
+                    </PageCaption>
+                    <PageCaption>
+                        of
+                    </PageCaption>
+                    <PageCaption number>
+                        {people.total_pages}
+                    </PageCaption>
+                </>
```

Comment:
Powtórzony kod z movies z jedyną zmianą że bierze dane z innego obiektu. Może lepiej zamknąć to w oddzielny komponent który przyjmie jako parametr albo people albo movies?

Reply:
Dobry pomysł, zdecydowanie do wdrożenia.